src/schemas/index.ts

```diff
@@ -0,0 +1,10 @@
+import * as types from '@nordicsemiconductor/lwm2m-types'
+
```

Comment:
Export `Config_50009` from `index`, this way all URNs can be imported from one single file.

Reply:
resolved in https://github.com/MLopezJ/asset-tracker-lwm2m-js/pull/7/commits/3bc425bc297bb23d0bafc9f9576d562c9751bc9e